settings_tab.py

```
@@ -0,0 +1,405 @@
+            )
+            return
+
+        # Always start from the shipped named-font values before layering the
+        # user's overrides. apply_fonts only touches attributes that are set
+        # (blank family or size<=0 are treated as "don't change"), so if the
+        # user clears just one field after previously customising both, the old
+        # value would otherwise stick — e.g. Arial/16 → blank family + 12
+        # would render as Arial/12 instead of system-default/12.
+        try:
+            ui_theme.reset_fonts_to_system(self.root)
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)